portfolio/src/main/webapp/script.js

```diff
@@ -137,15 +137,54 @@ function getMessage() {
 function handleResponse(response) {
   const textPromise = response.text();
   
-  textPromise.then(addMessageToDom);
+  textPromise.then(addSingleMessageToDom);
 }
 
 /**
- * Adds the given message to the DOM.
+ * Adds the given single message to the DOM.
  * @param {string} message The text to be added inside of the div 
        message-container. 
  */
-function addMessageToDom(message) {
+function addSingleMessageToDom(message) {
   const messageContainer = document.getElementById('message-container');
   messageContainer.innerHTML = message;
 }
+
+/**
+ * Adds multiple messages to the DOM as list elements.
+ * @param {object} messagesList An array containing messages.
+ */
+function addMultipleMessagesToDom(messagesList) {
```

Comment:
nit: "messagesList" can be renamed "messages".

Reply:
I originally had it as messagesList because I wanted to make it clear that it's a list object, but I think it is implied by using the plural of message.

Updated!